cmd2/rl_utils.py

```diff
@@ -70,7 +70,7 @@ class RlType(Enum):
     )
 
     # Check if we are running in a terminal
-    if sys.stdout.isatty():  # pragma: no cover
+    if sys.stdout and sys.stdout.isatty():  # pragma: no cover
```

Comment:
I would prefer this to be `if sys.stdout is not None ...` just to be clearer about what you're checking.